python/chronos/test/bigdl/chronos/__init__.py

```diff
@@ -13,3 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import pytest
+
+# framework
+op_torch = pytest.mark.torch
+op_tf2 = pytest.mark.tf2
+
+# distribution and automl
+op_automl = pytest.mark.automl
+op_distributed = pytest.mark.distributed
+
+# other mark
+op_tf1 = pytest.mark.tf1
+op_all = pytest.mark.all
```

Comment:
maybe `op_all` can be replaced by `tag1 or tag2 ...` or we may need to give all ut a `op_all` tag?

Reply:
If we give all UTs op_all tags, then we can move the torch to the ut method, sounds good.
Usage: 
* pytest -m "all"  <your_path>/file.py
* pytest -m "torch and distributed" <your_path>/file.py
* pytest -m "torch and automl" <your_path>/file.py
* pytest -m "torch and distributed and not onnxrt16" <your_path>/file.py

```python

@op_all
class Test():

    @op_torch
    @op_automl
    def test_method1():
        pass

    @op_torch
    @op_distributed
    def test_method2():
        pass

     @op_torch
     @op_distributed
     @op_onnxrt16
     def test_method3():
        passs